src/components/dialog/dialog.spec.js

```diff
@@ -168,6 +168,27 @@ describe('$mdDialog', function() {
       expect(content.id).toBe('dialogContent_' + dialog[0].id);
     }));
 
+    it('should not clobber the id from `md-dialog` when there is no content', inject(function ($mdDialog, $rootScope, $document) {
```

Comment:
LGTM except the end of this description "when there is no content". Shouldn't it say something like "when the user provides an ID"?